src/content/resources/architectural-overview.md

```diff
@@ -447,47 +447,61 @@ existing state objects when appropriate.
 So, if many widgets can contain state, how is state managed and passed around
 the system?
 
-As with any other class, you can use a constructor in a widget to initialize its
-data, so a `build()` method can ensure that any child widget is instantiated
-with the data it needs:
+As with any other class,
+you can use a constructor in a widget to initialize its data,
+so a `build()` method can ensure that any child widget
+is instantiated with the data it needs:
 
 ```dart
 @override
 Widget build(BuildContext context) {
-   return ContentWidget(importantState);
+   return ContentWidget(_importantState_);
```

Comment:
Are the underscores supposed to render like that?
<img width="526" alt="image" src="https://github.com/flutter/website/assets/111139605/25dde098-d66e-4200-b36e-0685767925bc">

Or is this a typo?
```suggestion
   return ContentWidget(importantState);
```

Reply:
They won't italicize in a code block. `[!importantState!]` will work.

Reply:
Yes, I learned that from Parker but have been in meetings and couldn't get to it. I've told him that italicizing and/or bolding code is fairly important to me.